### PR TITLE
Add P2P Video Chat Application to Extra Projects

### PR DESCRIPTION
**Pull request by karuppiah7890**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `staging` branch of FreeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes FreeCodeCamp/CurriculumExpansion#88

#### Description

Added a new application - P2P Video Chat application to the set of Extra Projects

### CODE REVIEW

**Comment by karuppiah7890** [comment]
The local test `npm test` failed. It was some problem with already written code and commits, because all the errors were not related to my changed.

**Comment by Greenheart** [comment]
@karuppiah7890 I really like this idea! :smile:

 @Em-Ant and @systimotic What do you think about this as a new project as one of the fullstack projects?

File: seed/challenges/10-extra-practice-projects/extra-practice-projects.json [hunk 1]
```diff
@@ -427,6 +427,32 @@
           ]
         }
       }
+    },
+    {
+      "id": "bd7150d8c442eddfafb5bd1c",
+      "title": "P2P Video Chat Application",
+      "description": [
+
```
Comment (raisedadead) [inline seed/challenges/10-extra-practice-projects/extra-practice-projects.json]: Remove Extra Line

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: seed/challenges/10-extra-practice-projects/extra-practice-projects.json [hunk 2]
```diff
@@ -427,6 +427,32 @@
...
+        "<strong>User Story:</strong> Anyone can create or join any room. And there can be any number of rooms, but all of them must have unique names. ",
+        "<strong>User Story:</strong> I can choose to not permit the site to access my microphone and webcam. If I choose not to do this, if some other driver problem occurs, I see an error message saying these are required. ",
+        "<strong>User Story:</strong> When I choose to cancel the room name input step, or if I type in no name, or just spaces, it should again ask me again to type in a valid room name. ",
+        "<strong>User Story:</strong> If one of the two people in the room get disconnected, they can reconnect to the same room and continue chatting.",
+        "Remember to use <a href='http://forum.freecodecamp.com/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck.",
+        "When you are finished, click the \"I've completed this challenge\" button and include a link to your Hyperdev App.",
+        "You can get feedback on your project by sharing it with your friends on Facebook."
+      ],
+      "challengeSeed": [],
```
Comment (raisedadead) [inline seed/challenges/10-extra-practice-projects/extra-practice-projects.json]: Can this be left blank?

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by karuppiah7890** [comment]
@Greenheart Thanks! :smile:

**Comment by Em-Ant** [comment]
@karuppiah7890 great project. I think it's very interesting. :+1:

**Comment by karuppiah7890** [comment]
@Em-Ant Thanks! I learned this in Google Code Labs actually. https://codelabs.developers.google.com/codelabs/webrtc-web and implemented using some npm modules instead of directly writing WebRTC code, which seemed a little bit of heavy lifting, with all the error handling and data channels stuff.

File: seed/challenges/10-extra-practice-projects/extra-practice-projects.json (hunk 2 above)
Comment (raisedadead) [inline seed/challenges/10-extra-practice-projects/extra-practice-projects.json]: Cool. thanks for the clarification

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
LGTM. thanks @karuppiah7890, sorry about the delay.

**Comment by karuppiah7890** [comment]
@raisedadead :+1:

**Pull request merged**